sites/next.skeleton.dev/src/examples/components/navigation/ExampleExpanded.svelte

```diff
@@ -16,27 +16,27 @@
 	<!-- Component -->
 	<Navigation.Rail expanded>
 		{#snippet header()}
-			<Navigation.Tile href="/" labelExpanded="Menu"><IconMenu /></Navigation.Tile>
+			<Navigation.Tile id="0" href="/" labelExpanded="Menu"><IconMenu /></Navigation.Tile>
 		{/snippet}
 		{#snippet tiles()}
-			<Navigation.Tile id="0" labelExpanded="Browse Files" href={hrefExample}>
+			<Navigation.Tile id="1" labelExpanded="Browse Files" href={hrefExample}>
```

Comment:
@phamduylong this is one of the only unique components we introduced that doesn't use Zag (intentionally). Now that I look at this, I wonder if rather than making ID mandatory, we utilize `useId()` to set a default automatically.

We didn't have access to `useId()` in Svelte when this component was built - that was introduced only when we integrated Zag. React has it built in as part of the library.

I think that'll provide a better experience - the ID will always be set, but users can still overwrite with a custom ID as they wish. Which would mean `id` would be optional again. Once less setting users have to deal with.

What do you think?